pkg/store/trtl/trtl_test.go

```diff
@@ -694,6 +694,107 @@ func (s *trtlStoreTestSuite) TestOrganizationStore() {
 	require.ErrorIs(err, storeerrors.ErrEntityNotFound)
 }
 
+func (s *trtlStoreTestSuite) TestContactStore() {
+	require := s.Require()
+
+	// Inject bufconn connection into the store
+	require.NoError(s.grpc.Connect(context.Background()))
+	defer s.grpc.Close()
+
+	// Connect a mock store
+	db, err := store.NewMock(s.grpc.Conn)
+	require.NoError(err)
+
+	// Make sure create errors with a nil contact
+	email, err := db.CreateContact(context.Background(), nil)
+	require.Empty(email)
+	require.Equal(err, storeerrors.ErrIncompleteRecord)
+
+	// Make sure create errors with a contact with an empty email
+	contact := &models.Contact{
+		Email: "",
+	}
+	email, err = db.CreateContact(context.Background(), contact)
+	require.Empty(email)
+	require.Equal(err, storeerrors.ErrIncompleteRecord)
+
+	// Create a valid contact
+	contact = &models.Contact{
+		Email:      "testemail",
+		Vasps:      []string{"foo", "bar"},
+		Verified:   false,
+		Token:      "testtoken",
+		VerifiedOn: "",
+	}
+	email, err = db.CreateContact(context.Background(), contact)
+	require.Equal(email, "testemail")
+	require.NoError(err)
```

Comment:
Can you add a quick check to ensure that after the contact is created the created/modified timestamps are set? And after update only `modified` is modified?